### PR TITLE
Style page header typography

### LINKED ISSUE #33: Component: Page Header
- [x] Page Title
- [x] Page Excerpt on Archive, Resources, and Glossary pages

### PR DESCRIPTION
**Pull request by jnschrag**
Closes #33 by creating the page header component. This styles the page title & description (if one exists).

### CODE REVIEW

File: _includes/page-header.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <section class="page__header">
-Page Header
+  <h1 class="page__title">{{ page.title | escape }}</h1>
+  {% assign content = content | strip_newlines %}
+  {%- if content != '' -%}
+  <div class="page__desc">{{ content }}</div>
```
Comment (leezaweb) [inline _includes/page-header.html]: If the glossary is using this layout, then the page description might be repeated. 
you might do {% if page.name != 'glossary' %} {{ content }} {% endif %} on the pages.html 
or here, do
 {% if page.name != 'glossary' %} <div class="page__desc">{{ content }}</div> {% else %} <div class="page__desc">{{ site.data.language.glossary_description }}</div> {% endif %}

Comment (jnschrag) [reply _includes/page-header.html]: Can you push the glossary branch tomorrow so I can look at it? It's unclear to me why the glossary page wouldn't be using the page header component, so there shouldn't be a conflict. The description should be the "content" section of the markdown page (as you pointed out in the other PR).

**Comment by jnschrag** [comment]
@leezaweb Let me talk to Tucker about what she wants to do with the page header. This is the same issue I had that caused me to make the 850px breakpoint for the page structure. Tablet size isn't supposed to have padding around the title, but that doesn't work quite as well on the archive pages when they're flush to the sides (unlike the posts, which are not).

File: _includes/page-header.html (hunk 1 above)
Comment (leezaweb) [reply _includes/page-header.html]: I should probably rephrase my question. It's mostly concerning the glossary. Where should the glossary description go?

**Comment by jnschrag** [comment]
This has been updated to account for the new breakpoints.

@aleesteele The padding around the posts will be taken care of in the post-block styling. The reason I had to strip_newlines for the content is to account for what happens if a file is empty but has empty lines.

screen shot 2018-11-06 at 9 55 14 am

Without it, this ^^ would have registered as there being content, even though there isn't any.

**Pull request merged**